From f3e939eb9b6051578b59950b1c3f0f441e917f88 Mon Sep 17 00:00:00 2001
From: Seth Jennings <sjenning@redhat.com>
Date: Mon, 18 Aug 2014 15:42:48 -0400
Subject: [edac] sb_edac: Fix mix tab/spaces alignments

Message-id: <1408376571-8244-10-git-send-email-sjenning@redhat.com>
Patchwork-id: 87869
O-Subject: [RHEL7.1 PATCH 09/12] sb_edac: Fix mix tab/spaces alignments
Bugzilla: 1127019
RH-Acked-by: Aristeu Rozanski <aris@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Mauro Carvalho Chehab <m.chehab@samsung.com>

Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7802951
Bugzilla: https://brewweb.devel.redhat.com/taskinfo?taskID=7802951
Upstream: c41afdca29acfdc78b89f936cdcb8cabc6b08aae

We should not have spaces before ^I on alignments.

Signed-off-by: Mauro Carvalho Chehab <m.chehab@samsung.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/edac/sb_edac.c b/drivers/edac/sb_edac.c
index a5ef406..af84442 100644
--- a/drivers/edac/sb_edac.c
+++ b/drivers/edac/sb_edac.c
@@ -311,7 +311,7 @@ struct sbridge_channel {
 };
 
 struct pci_id_descr {
- int    dev_id;
+ int   dev_id;
  int   optional;
 };
 
@@ -946,7 +946,7 @@ static int get_memory_error_data(struct mem_ctl_info *mci,
  struct mem_ctl_info *new_mci;
  struct sbridge_pvt *pvt = mci->pvt_info;
  struct pci_dev  *pci_ha;
- int    n_rir, n_sads, n_tads, sad_way, sck_xch;
+ int   n_rir, n_sads, n_tads, sad_way, sck_xch;
  int   sad_interl, idx, base_ch;
  int   interleave_mode;
  unsigned  sad_interleave[pvt->info.max_interleave];
@@ -1348,7 +1348,7 @@ static int sbridge_get_onedevice(struct pci_dev **prev,
  * sbridge_get_all_devices - Find and perform 'get' operation on the MCH's
  *        devices we want to reference for this driver.
  * @num_mc: pointer to the memory controllers count, to be incremented in case
- *      of success.
+ *     of success.
  * @table: model specific table
  *
  * returns 0 in case of success or error code
-- 
1.7.1